new_arrivals_chi/app/main.py

```diff
@@ -23,7 +23,7 @@
 
 from flask import Flask, Blueprint, render_template, request
 
-main = Blueprint("main", __name__)
+main = Blueprint("main", __name__, static_folder="/static")
```

Comment:
Added a static folder so that you could have CSS and JS files external to your HTML files.